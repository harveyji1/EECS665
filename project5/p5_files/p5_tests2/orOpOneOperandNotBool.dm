a : bool;
b : int;
c : bool;

main : () int {
	a = false;
	b = 2;
	c = a or b; 
}

