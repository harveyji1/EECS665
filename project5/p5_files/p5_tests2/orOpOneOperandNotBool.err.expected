FATAL [8,11]-[8,12]: Logical operator applied to non-bool operand
Type Analysis Failed
